tests/test_basic_int.py

```diff
@@ -0,0 +1,27 @@
+import h3.api.basic_int as h3
+
+
+def test1():
```

Comment:
I'd bias toward a more descriptive test name, like `test_geo_to_h3`

I'd also group your tests by the function being tested rather than input type being tested. Might be nice to use a [test table](https://github.com/golang/go/wiki/TableDrivenTests) for that

Reply:
Added some better names. :)

Most of the testing is through the `str` interface. Since the underlying code paths are the same, I just have this test here to check that the basic conversion functionality works, which should imply that all the other tests will work.

It's true that table testing would be more robust. I'll add a task for us to take a look at this in the future.

Reply:
issue: https://github.com/uber/h3-py/issues/102